examples/tokenvm/scripts/run.sh

```diff
@@ -44,22 +44,20 @@ if ${UNLIMITED_USAGE}; then
 fi
 
 echo "Running with:"
-echo AGO_LOGLEVEL: ${AGO_LOGLEVEL}
-echo LOGLEVEL: ${LOGLEVEL}
+echo AGO_LOGLEVEL: "${AGO_LOGLEVEL}"
+echo LOGLEVEL: "${LOGLEVEL}"
 echo VERSION: ${VERSION}
```

Comment:
I find it surprising the other items here don't require quotes?

Reply:
IMO the consistency is preferred. If you can add that option, that would be great?